Sources/SwiftProtobuf/Google_Protobuf_Timestamp_Extensions.swift

```diff
@@ -224,17 +207,30 @@ private func formatTimestamp(seconds: Int64, nanos: Int32) -> String? {
     // The following is crude, but it works.
     // TODO: If String(format:) works, that might be even better
```

Comment:
Is this still true?  https://github.com/apple/swift-protobuf/blob/5702ce39fa52d950479024b21f6236bfa938566e/Performance/Harness.swift#L103 is using it.